resource_customizations/kubevirt.io/VirtualMachine/health.lua

```diff
@@ -0,0 +1,18 @@
+hs = { status="Progressing", message="No status available"}
+if obj.status ~= nil then
+  if obj.status.ready ~= nil then
```

Comment:
Note that `.status.ready` and `.status.created` are boolean field, so you should actually be looking at their values, and not their presence.